src/components/LogInForm/LogInForm.js

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import SubmitButton from "common/buttons/SubmitButton.js";
+import FormContainer from "common/forms/FormContainer";
+import PasswordInput from "common/forms/PasswordInput";
+import TextInput from "common/forms/TextInput";
+import useField from "hooks/useField";
 import { useDispatch } from "react-redux";
-import { useNavigate } from 'react-router-dom';
-import { FormContainer, PasswordInput, SubmitButton, TextInput } from '../../formElements';
-import useField from '../../hooks/useField';
-import { logUserIn } from '../../reducers/loggedInUserReducer';
-import loginService from '../../services/login';
+import { useNavigate } from "react-router-dom";
+import { logUserIn } from "reducers/loggedInUserReducer";
+import loginService from "services/login";
 
 const LogInForm = () => {
   const navigate = useNavigate();
-  const name = useField('text');
-  const password = useField('password');
+  const name = useField("text");
+  const password = useField("password");
   const dispatch = useDispatch();
   const handleLogIn = async (event) => {
-    event.preventDefault()
+    event.preventDefault();
     const user = {
       name: name.value,
       password: password.value,
-    }
+    };
 
     try {
       const userToLogIn = await loginService.login(user);
-      dispatch(logUserIn(userToLogIn))
+      dispatch(logUserIn(userToLogIn));
     } catch (error) {
-      window.alert('Login failed')
+      window.alert("Login failed");
```

Comment:
In that react-hook-form task, refactor here to use react toast like https://react-hot-toast.com/